framework/SupervisedLearning/ARMA.py

```diff
@@ -332,6 +499,8 @@ def _computeNumberOfBins(self,data):
     # tend towards too many bins, not too few
     # also don't use less than 20 bins, it makes some pretty sketchy CDFs otherwise
     n = max(int(np.ceil((max(data) - min(data))/size)),20)
+    n *= 100
```

Comment:
is it hard coded value?

Reply:
This is a hard-coded scaling factor, yes, similar to how the line above is hard-coded algorithm.